packages/enzyme/src/ReactWrapper.js

```diff
@@ -374,9 +377,16 @@ class ReactWrapper {
    * @returns {Boolean}
    */
   contains(nodeOrNodes) {
+    const adapter = getAdapter(this[OPTIONS]);
+
     const predicate = Array.isArray(nodeOrNodes)
-      ? other => containsChildrenSubArray(nodeEqual, other, nodeOrNodes)
-      : other => nodeEqual(nodeOrNodes, other);
+      ? other => containsChildrenSubArray(
+        nodeEqual,
+        other,
+        nodeOrNodes.map(adapter.elementToNode),
```

Comment:
Let’s make this use an arrow function, so adapters only get the single argument

Reply:
I also see this is how it already is in https://github.com/airbnb/enzyme/pull/1423/files#diff-fa27c82ee4fde075bae6173848e17c82R448; let’s change both

Reply:
> Let’s make this use an arrow function, so adapters only get the single argument
 
Do you mean the `nodeOrNodes` `map` callback should be an arrow function?

Reply:
yep! like `.map(x => adapter.elementToNode(x))`

Reply:
Done 🙂